scripts/lib/install

```diff
@@ -276,7 +276,7 @@ EOF
     #
     # (There are more possibilities in the case of dpkg errors.)  Here
     # we are checking for either empty or not-installed.
-    if [ -n "$TRAVIS" ] || ! dpkg-query --showformat '${Status}\n' -W rabbitmq-server 2>/dev/null | grep -vq ' not-installed$'; then
+    if [ -n "$CIRCLECI" ] || ! dpkg-query --showformat '${Status}\n' -W rabbitmq-server 2>/dev/null | grep -vq ' not-installed$'; then
```

Comment:
Did you try deleting the TRAVIS check here?  I think the `TRAVIS` conditional in this block is just a workaround for Travis having already had rabbitmq-server installed, and might be removable entirely.

Reply:
@timabbott I tried doing that but after applying puppet configurations rabbitmq could not start.
throwing error like these.
```
ERROR: node with name "rabbit" already running on "a4721d3c1f9d"
```
starting rabbitmq after installation solved this issue for now
All the issues in error logs are also gone.